styleguide/source/_patterns/03-organisms/footer.twig

```diff
@@ -0,0 +1,3 @@
+<footer id="footer" class="ama__footer" role="contentinfo">
+  {% include '@base/placeholder.twig' with {'placeholder':{'text':'Footer'}} %}
+</footer>
```

Comment:
Do we want to include global items in layouts? Those are unlikely to change as the layouts change and therefore seem like there is less value in continuing to add these items over and over.

(CC: @nstriedinger @chelsiejohnston)